go/client/wallet_api_doc.go

```diff
@@ -28,14 +28,17 @@ Set the inflation destination for an account to some other account:
 Set the inflation destination for an account to itself:
     {"method": "set-inflation", "params": {"options": {"account-id": "GDUKZH6Q3U5WQD4PDGZXYLJE3P76BDRDWPSALN4OUFEESI2QL5UZHCK4", "destination": "self"}}}
 
-Send XLM to a keybase user (there is no confirmation so be careful):
+Send XLM to a Keybase user (there is no confirmation so be careful):
     {"method": "send", "params": {"options": {"recipient": "patrick", "amount": "1"}}}
 
-Send $10 USD worth of XLM to a keybase user:
+Send $10 USD worth of XLM to a Keybase user:
     {"method": "send", "params": {"options": {"recipient": "patrick", "amount": "10", "currency": "USD", "message": "here's the money I owe you"}}}
 
-If you send XLM to a keybase user who has not established a wallet yet, you can 
-cancel the payment before the recipient claims it and the XLM will be returned 
+Send 10 AnchorUSD to a Keybase user as a path payment by converting XLM:
```

Comment:
can you mention there is no confirmation (like in line 31).

Reply:
no limit no confirmation poker is a trap api. How about a required upper limit on the amount of source asset? Or an endpoint to calculate the path or limit.

Reply:
@mlsteele I'll do both!